app/assets/css/repl.css

```diff
@@ -1,11 +1,11 @@
-.textarea,
-input[type='search'],
-pre,
-textarea {
+.repl .textarea,
+.repl input[type='search'],
+.repl pre,
+.repl textarea {
     box-shadow: none;
 }
 
-textarea {
+.repl textarea {
     background: none;
     border: 0;
     color: #2a2734;
```

Comment:
var needed?

Reply:
For colors that are only used once, I just left them hardcoded